ansible/roles/sw-cli-tools/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: download sw cli tools
+  get_url:
+    url: http://shopwarelabs.github.io/sw-cli-tools/sw.phar
+    dest: ~/bin/sw
+    mode: 0755
+  tags: sw-cli-tools
+
+- name: create symlink for cw cli tools
```

Comment:
The symlink should not be necessary since `~/bin/` is already in the shopware users $PATH.